.github/Pull_Request_template.md

```diff
@@ -0,0 +1,15 @@
+---
+If screen-shot availabel: ' '
+SS-1: Here
+SS-2: Here
+
+---
```

Comment:
I would exclude screen-shot related option as 99.9% of commits to daal4py are not related to any sort of UI, so it have to be removed by all other contributors